hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/om/lock/TestKeyPathLock.java

```diff
@@ -136,8 +136,7 @@ private void testSameKeyPathWriteLockMultiThreadingUtil(
     // Now all threads have been instantiated.
     Assert.assertEquals(0, countDownLatch.getCount());
 
-    lock.acquireWriteHashedLock(resource,
-        generateResourceHashCode(resource, sampleResourceName));
+    lock.acquireWriteLock(resource, sampleResourceName);
```

Comment:
Nit: can we add `countDown.await()` similar to what is done under `TestKeyPathLock#testDiffKeyPathWriteLockMultiThreadingUtil`:
https://github.com/apache/ozone/blob/1f57ff6ea7af532fdfe5fc8b1a11a5c53f744e43/hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/om/lock/TestKeyPathLock.java#L127-L134

Reply:
similar to the above.